cmd/lncli/commands.go

```diff
@@ -1750,13 +1777,19 @@ func sendPayment(ctx *cli.Context) error {
 	// If a payment request was provided, we can exit early since all of the
 	// details of the payment are encoded within the request.
 	if ctx.IsSet("pay_req") {
+		if !ctx.IsSet("force") {
```

Comment:
I think this should be `!ctx.Bool("force")`, as now it would skip the confirmation even when setting `--force=0`.